src/map.cpp

```diff
@@ -2601,6 +2601,20 @@ bool map::has_nearby_table( const tripoint &p, int radius )
     return false;
 }
 
+bool map::has_nearby_chair( const tripoint &p, int radius )
+{
+    for( const tripoint &pt : points_in_radius( p, radius ) ) {
+        const optional_vpart_position vp = veh_at( p );
```

Comment:
```suggestion
        const optional_vpart_position vp = veh_at( pt );
```
The assignment should also be move down. It won't be need in case `has_flag` returns true.